framework/src/systems/SystemBase.C

```diff
@@ -134,13 +137,27 @@ SystemBase::getFieldVariable(THREAD_ID tid, const std::string & var_name)
   return *_vars[tid].getFieldVariable<T>(var_name);
 }
 
+template <typename T>
+MooseVariableField<T> &
+SystemBase::getActualFieldVariable(THREAD_ID tid, const std::string & var_name)
+{
+  return *_vars[tid].getActualFieldVariable<T>(var_name);
+}
+
 template <typename T>
 MooseVariableFE<T> &
 SystemBase::getFieldVariable(THREAD_ID tid, unsigned int var_number)
 {
   return *_vars[tid].getFieldVariable<T>(var_number);
```

Comment:
`getFEVariable`?

Reply:
What is your question here? That we should consider renaming the method? Then we have to do the beautiful song and dance of going and fixing all the apps which is excruciating